### PR TITLE
feat(ButtonGroup): added top alignment to icon in ButtonGroup

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

[#1572](https://github.com/carbon-design-system/ibm-dotcom-library/issues/1572)

### Description

Updated ButtonGroup so that the text and icon are top aligned

Screen Shot 2020-06-16 at 4 24 03 PM

### Changelog

**New**

- added CSS styling

### CODE REVIEW

File: packages/react/src/components/ButtonGroup/ButtonGroup.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ const ButtonGroup = ({ buttons, enableSizeByContent }) => {
               ),
               item => {
                 item.style.width = `${width + 1}px`;
+                item.style.display = `inline-block`;
```
Comment (asudoh) [inline packages/react/src/components/ButtonGroup/ButtonGroup.js:47]: Given `display: inline-block` doesn't need JavaScript-based calculation, would you move this to `_buttongroup.scss`?

**Comment by jeffchew** [comment]
Hey @ariellalgilmore sorry I might have missed this during your ramp up, we have configured our PRs to build based on the label set. When setting `package: react`, that ends up building the react deploy preview. There are others as well, but that's pretty much the main one.

**Comment by andysherman2121** [comment]
Left a comment on the Percy build. Might need a fix on one of the stories there

**Comment by jeffchew** [comment]
@ariellalgilmore looks like some alignment issues with some of the button usage, can you take a look? https://percy.io/IBM-Digital-Design/IBM.com-Library-React/builds/5718600?utm_campaign=IBM-Digital-Design

**Comment by jeffchew** [comment]
@ariellalgilmore the alignment still looks a little off, it's off centered by a little bit it seems.

**Pull request merged**